pontoon/static/js/tags/admin/table.js

```diff
@@ -0,0 +1,42 @@
+
+import React from 'react';
+
+import CheckboxTable from 'widgets/tables/checkbox';
+
+
+export default class TagResourceTable extends React.PureComponent {
+
+    get columnResource () {
+        return {
+            Header: "Resource",
+            id: "type",
+            Cell: this.renderResource};
+    }
+
+    get columns () {
+        return [this.columnResource];
+    }
+
+    get submitClass () {
+        return "tag-resources-associate";
+    }
```

Comment:
Why are these three getters? It would be much simpler to make them attributes.

Reply:
yep, ill update

Reply:
i think only this one can be moved to an attr as the others use `this`

Reply:
re js `Class.attrs` im remming the thing that is a pita with these - you cant do python-like...

```
Class A(object):
     foo = "FOO"

class B(A):
    
     @property
     def foo(self):
          return "MANGLED FOO"

```

altho im not clear whether this is an es6+ problem, or whether its just the way webpack is currently set